requirements.txt

```diff
@@ -31,3 +31,5 @@ scipy
 scikit-learn==1.2.2
 pynvml
 art
+wandb
+fschat
```

Comment:
we should set version for fschat in case a change upstream causes a change with prompting without our knowledge.

Also, I am not quite a fan of installing an entire package just for their conversation script, but it seems like a clean integration.